charts/stable/cloudflareddns/docs/validation.md

```diff
@@ -0,0 +1,29 @@
+# Input Validation
+
+**`CF Hosts`**
+Not accepted domains are:
+
+- String (eg. `example.cf`)
+- String (eg. `example.ga`)
+- String (eg. `example.gq`)
+- String (eg. `example.ml`)
+- String (eg. `example.tk`)
+
+Regex used to match this: `^((?!\.cf$|\.ga$|\.gq$|\.ml$|\.tk$).)*$`
+You can try live [here](https://regex101.com/r/GFEGVB/1)
```

Comment:
![image](https://user-images.githubusercontent.com/47820033/210730763-c94347b7-aa81-4942-8d7a-454bb07c6c4d.png)

Not as advertised :P
It accepts the `.tk`

Reply:
thats do to the `;` causing it to act as a new tld

Reply:
if possible, i was planning on creating a list for both and add the validation to the single item itself.
CF_ZONE doesnt just accept tlds domains but also zone ids too. the validation does work to that extent